presidio-analyzer/presidio_analyzer/conf/default_recognizers.yaml

```diff
@@ -2,6 +2,26 @@ supported_languages:
   - en
 
 recognizers:
+  # Recognizers listed here can either be loaded from the recognizers defined in code (type: predefined),
```

Comment:
As discussed, added a comment explaining how to use this file.

Reply:
Unless I missed something, the implementation for this specific recognizer had also it & pl as languages.